azurerm/resource_arm_container_group.go

```diff
@@ -277,6 +309,10 @@ func resourceArmContainerGroupRead(d *schema.ResourceData, meta interface{}) err
 	}
 	flattenAndSetTags(d, resp.Tags)
 
+	if imageRegCreds := flattenContainerImageRegistryCredentials(d, resp.ImageRegistryCredentials); imageRegCreds != nil {
+		d.Set("image_registry_credential", imageRegCreds)
```

Comment:
Could we check for an error here? like:

```go
if err := d.Set("capabilities", flattenAzureRmCosmosDBAccountCapabilities(resp.Capabilities)); err != nil {
		return fmt.Errorf("Error setting `capabilities`: %+v", err)
	}
```

`d.Set()` sill handle a nil value correctly

Reply:
Sure, make sense I'll fix this up